src/main.cpp

```diff
@@ -938,14 +945,14 @@ static bool auto_select_world(GameParams *game_params)
 	// If there is only a single world, use it
 	if (worldspecs.size() == 1) {
 		world_path = worldspecs[0].path;
-		dstream <<_("Automatically selecting world at") << " ["
+		dstream << "Automatically selecting world at ["
 		        << world_path << "]" << std::endl;
 	// If there are multiple worlds, list them
 	} else if (worldspecs.size() > 1 && game_params->is_dedicated_server) {
-		std::cerr << _("Multiple worlds are available.") << std::endl;
-		std::cerr << _("Please select one using --worldname <name>"
-				" or --world <path>") << std::endl;
-		print_worldspecs(worldspecs, std::cerr);
+		rawstream << "Multiple worlds are available.\n"
```

Comment:
Why `std::endl` in one case and `\n` in another?

Reply:
`endl` flushes the stream. No need to do that multiple times if we're printing a bunch of text.
(though I think with our current logging infrastructure there is no difference)